androidApp/src/main/java/dev/johnoreilly/confetti/bookmarks/BookmarksView.kt

```diff
@@ -156,17 +161,28 @@ private fun BookmarksHorizontalPager(
             contentPadding = WindowInsets.safeDrawing.only(WindowInsetsSides.Bottom)
                 .asPaddingValues()
         ) {
-            items(displayedSessions) { session ->
-                SessionItemView(
-                    conference = conference,
-                    session = session,
-                    sessionSelected = navigateToSession,
-                    isBookmarked = bookmarks.contains(session.id),
-                    addBookmark = { sessionId -> addBookmark(sessionId) },
-                    removeBookmark = { sessionId -> removeBookmark(sessionId) },
-                    onNavigateToSignIn = onSignIn,
-                    user = user,
-                )
+            val sessionsByDate = displayedSessions.groupBy { it.startsAt }
```

Comment:
Can be something we do in follow up PR if needed but wonder if we should do this in the view model....that way iOS client for example can also use data directly then (when we add bookmarks to it that is!)

Reply:
Yes, I think it would be better to place this in the VM, and from there the consumers will receive the data already grouped, regardless of where the consumer is (which platform).

Reply:
Does it make sense to have past sessions also grouped by date?

Reply:
@joreilly  I decided to already provide the grouped type from the ViewModel as you suggested.